bin/run-canonical-extensions-tests.sh

```
@@ -0,0 +1,106 @@
+	workflow=$( gh workflow list --json path,id --repo $repository | jq --compact-output '.[]' | grep -E '.github/workflows/(manual-ci.yml|ci-manual.yml)' )
+	workflow_path=$( echo $workflow | jq --raw-output '( .path )' )
+	workflow_id=$( echo $workflow | jq --raw-output '( .id )' )
+	echo -n " workflow ${workflow_path} (#${workflow_id})"
+
+	# Report last run details.
+	previous_run=$( gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/${repository##https://github.com/}/actions/workflows/${workflow_id}/runs?per_page=1 --jq '.workflow_runs.[].id' )
+	echo -n " previous run #${previous_run} "
+
+	# Start a new run and report back.
+	echo "{\"wc-version\":\"$version\", \"wp-version\":\"$wordpress\",  \"php-version\":\"$php\", \"qit-tests\":\"WooCommerce Pre-Release Tests (includes Activation, WooCommerce E2E and API tests)\"}" | gh workflow run ${workflow_id} --json --repo $repository >/dev/null
+	for i in {1..10}; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)